.gitignore

```diff
@@ -0,0 +1 @@
+github.com
```

Comment:
just curious where this comes from - As a gitignore basis I tend to use https://www.toptal.com/developers/gitignore/

Reply:
I'm not sure why this particular value is there. It came from this commit https://github.com/libp2p/go-libp2p-kad-dht/commit/54f20b5c0f8e408d5ca07cbb6a9c12c98798cbbd